packages/common/src/types/errors.ts

```diff
@@ -0,0 +1,13 @@
+import { RpcError } from '@protobuf-ts/runtime-rpc';
+
+export { RpcError };
+
+export function isRpcError(error: unknown): error is RpcError {
+    const err = error as RpcError;
+    return (
+        err.code !== undefined &&
+        err.meta !== undefined &&
+        err.name !== undefined &&
+        err.toString !== undefined
+    );
```

Comment:
```suggestion
    return error instanceof RpcError;
```

Since `RpcError` is a class, this should also work, and would be more precise as well

Reply:
Much better, I'll test it and merge it, okay?

Reply:
Sure 😄